charts/kong/templates/migrations-post-upgrade.yaml

```
@@ -58,6 +58,10 @@ spec:
         {{- include "kong.volumeMounts" . | nindent 8 }}
       securityContext:
       {{- include "kong.podsecuritycontext" . | nindent 8 }}
+      {{- if .Values.nodeSelector }}
+      nodeSelector:
+      {{ toYaml .Values.nodeSelector | indent 8 }}
```

Before this PR, all 3 edited files prefer `nindent` (with `{{-`) to `indent`ing - this PR appears to be the first introduction of `indent`.

Is there a rationale for breaking consistency? If not, could we do:

```suggestion
      {{- toYaml .Values.nodeSelector | nindent 8 }}
```
throughout this PR?

---

Not really. Copy-pasta from the deployment template; newline-chomping convention isn't consistent throughout the templates.